lib/utils.js

```diff
@@ -117,6 +120,27 @@ async function checkAppPresent (app) {
   log.debug('App is present');
 }
 
+async function getGitRev () {
```

Comment:
This function will not do what you want it to, I don't think. It will, if it returns anything, return the containing project (i.e., run through the full Appium project, it will give the get revision for the Appium project, not the driver).